core-common/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -85,6 +86,28 @@
     String componentModel() default "default";
 
     /**
+     * Specifies the name of the implementation class. A single {@code *} wildcard can be used and will be replaced
+     * by the interface's or abstract class' name.
+     * <p>
+     * Defaults to {@code *Impl} which means: <i>postfix the name with {@code Impl}</i>
+     *
+     * @return The implementation suffix.
+     * @see #implPackage()
+     */
+    String implName() default DEFAULT_IMPL_NAME;
```

Comment:
Could we avoid the abbreviations in the public API, i.e. `implementationName` and `implementationPackage`. Reads a bit nicer.